pkg/webhook/resources/setting/validator.go

```diff
@@ -228,12 +231,20 @@ func (v *settingValidator) validateBackupTarget(setting *v1beta1.Setting) error
 
 	vmBackups, err := v.vmBackupCache.List(metav1.NamespaceAll, labels.Everything())
 	if err != nil {
-		return werror.NewInternalError(err.Error())
+		return werror.NewInternalError(fmt.Sprintf("Can't list VM backups, err: %+v", err.Error()))
 	}
 	if hasVMBackupInCreatingOrDeletingProgress(vmBackups) {
 		return werror.NewBadRequest("There is VMBackup in creating or deleting progress")
 	}
 
+	vmRestores, err := v.vmRestoreCache.List(metav1.NamespaceAll, labels.Everything())
+	if err != nil {
+		return werror.NewInternalError(fmt.Sprintf("Failed to get the VM restore objects, err: %+v", err.Error()))
+	}
+	if hasVMRestoreInCreatingOrDeletingProgress(vmRestores) {
+		return werror.NewBadRequest("Can't update the backup target, has a VM either in the process of creating or deleting")
```

Comment:
`has a VM either ...` --> `there is a VM either ...` sounds better?

Reply:
Yes, sounds good to me. How about `Can't update the backup target. There is a VM either in the process of creating or deleting`

Reply:
agree 👍🏼 , looks good!

Reply:
Updated it. Thank you.